.github/ISSUE_TEMPLATE/certification_request.yaml

```diff
@@ -0,0 +1,57 @@
+name: Request for CAPI Provider Certification
+description: Request that a CAPI provider is added to the list of certified providers.
+body:
+  - type: textarea
+    id: provider
+    attributes:
+      label: What is the provider you would like to be added?
+      placeholder: "Specify the name and the type of provider."
+    validations:
+      required: true
+
+  - type: textarea
+    id: repository
+    attributes:
+      label: What is the repository URL?
+      placeholder: "Link to GitHub repository where the provider's source code is hosted."
+    validations:
+      required: true
+
+  - type: textarea
+    id: check_successful
+    attributes:
+      label: Did you follow the certification process?
+      placeholder: "You need to follow the certification process before requesting a provider is added to the list. You can refer to the Turtles documentation."
+    validations:
+      required: true     
+
+  - type: textarea
+    id: certification_info
+    attributes:
+      label: Link to the test suite integration for the provider.
+      placeholder: "Specify where we can find the integration with Turtles' test suite to validate the provider."
+    validations:
+      required: true
+
+  - type: textarea
+    id: why
+    attributes:
+      label: Why do you think this provider should be added?
+      placeholder: "All providers in the certified list are actively validated which means we have to keep it limited to the most relevant projects."
+    validations:
+      required: true
+
+  - type: textarea
+    id: additional
+    attributes:
+      label: Anything else you would like to add?
+      placeholder: "Miscellaneous information."
+    validations:
+      required: false
+
+  - type: textarea
+    id: templateLabel
+    attributes:
+      label: Label(s) to be applied
+      value: |
+        /kind feature
```

Comment:
Is this needed? We don't have a prow running anyway and `/kind feature` will have no effect.